src/bundle/Resources/public/scss/_custom.scss

```diff
@@ -218,3 +206,8 @@ $input-focus-bg: transparent;
 $input-focus-border-color: $ibexa-color-primary;
 $input-focus-color: $ibexa-color-font-pale;
 $input-focus-box-shadow: 0 0 calculateRem(2px) 0 rgba($ibexa-color-primary, 0.75);
+
+$btn-font-size: calculateRem(13px);
```

Comment:
I guess these are bootstrap variables, what would you say on moving them to some other file and here leave only our own variables? @dew326